### PR TITLE
Finished Wumpus

### PR DESCRIPTION
**Pull request by 1118422ILHS**

### CODE REVIEW

File: Core/WumpusCore/Wumpus/Wumpus.cs [hunk 1]
```diff
@@ -7,44 +8,65 @@ namespace WumpusCore.Wumpus
...
         }
+        
         /// <summary>
-        /// Changes state and starts minigame, then changes state again based on result of battle.  
-        /// Better description coming soon
+        /// Sets the state of the Wump
         /// </summary>
-        public void startBattle()
+        /// <param name="newState">The new state of the Wump.  0 for sleep, 1 for fleeing arrow shoot, 2 for fleeing combat, 3 for in combat, 4 for ded</param>
+        public void setState(int newState)
```
Comment (roguefirework) [inline Core/WumpusCore/Wumpus/Wumpus.cs:63]: This can take a state maybe?

File: Core/WumpusCore/Wumpus/Wumpus.cs [hunk 2]
```diff
@@ -7,44 +8,65 @@ namespace WumpusCore.Wumpus
...
+        {
+            this.WumpState = (State)newState;
+        }
+        public int getWumpState()
```
Comment (roguefirework) [inline Core/WumpusCore/Wumpus/Wumpus.cs:67]: This can return a state not an int

**Review by roguefirework — CHANGES_REQUESTED** [review]

**Review by roguefirework — APPROVED** [review]
Looks good

**Review by MaximumMaxxx — APPROVED** [review]
I agree

**Pull request merged**